components/chips/chips.js

```diff
@@ -0,0 +1,34 @@
+const lastInputChip = document.querySelector("#demo-chip-container");
+const chipInputElement = document.querySelector("#chip-inputElement");
+const inputChipConatinerDiv = document.querySelectorAll(".chips-container")[1];
+let n = 2;
+
+function createChip(chipLabel) {
+
+    const div = document.createElement("div");
+    div.setAttribute("class", "chip-body black-chip")
+    const span = document.createElement("span");
+    span.innerText = chipLabel;
+    const i = document.createElement("i");
+    i.setAttribute("class", "fas fa-times close");
+
+    div.appendChild(span);
+    div.appendChild(i);
```

Comment:
This is a good way, but we could also do something like this which does the work in fewer lines and also we don't have to worry about the hassle of setAttribute and createElement and all that stuff...

```javascript
const chipElement = 
    `
        <div class="chip-body black-chip">
             <span>chipLabel</span>
             <i class="fas fa-times close"></i>
        </div> 
    `
return chipElement;
```

Reply:
Hey, Vishal thanks a lot for you feedback, will look into this.

Reply:
Hey Vishal if we try to implement it in this way then i guess we have to use innerText later, which i have read somewhere using it is a bad practice. I did it in this way first, then again i changed it to the createElement and setattribute one.

Reply:
You might be but I also did in a very similar way as I mentioned and there I used `innerHTML`. You can check [this](https://codepen.io/VishalPatil18/pen/poWXpez?editors=0010) for reference.

Reply:
Thanks @VishalPatil18 will look into it.